packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -69,9 +74,9 @@ describe("base-radio", () => {
 
         // Assert
         expect(
-            screen.getByRole("checkbox", {name: "Select Choice E"}),
+            screen.getAllByRole("button", {hidden: true})[4],
```

Comment:
I see this throughout the PR, but `{hidden: true}` seems like a red flag - especially when we're replacing a named element like in this case. One of the goals of Testing-Library is to recreate the behavior of a user. Take this example:

1. User sees a checkbox labeled "Select Choice E"
2. There's n invisible buttons and we want button 5

The first one describes something a user would interact with and the other relies heavily on the structure of code. I'm sure you thought of this when you made the change, but I would ask that we try again to see if we can do it closer to how it was originally; if we can't, that might be a sign that things aren't as accessible as we think.

Reply:
Not being able to `getByRole("checkbox", ...)` worries me as well.  The wonder-blocks form elements weren't designed to allow this level of customization, but... if we want to wonder-blocks-ify perseus while maintaining the design of the radios and checkboxes, then I think we should look at what changes to wonder-blocks-form would be necessary to support that.  If there are time constraints about when we need to complete this accessibility work by then we may not have time to update wonder-blocks-form at the moment.

Reply:
@kevinbarabash for content I think that `getByRole("checkbox"...` would work here to reach the input experience, but she's testing the mouse interface button.

As far as incorporating the wonder blocks radio - I think that's a great plan for the future but that wasn't the plan for this piece of work - this was taking what we already had and improving the a11y interface. We need to release and update perseus in the near future and I think using the wonder blocks component might trigger conversations and work that would push that release out.

Reply:
Will confirm clicking input (visually hidden screen reader choice) is the same as clicking the button (visually available but ARIA hidden choice), and then I will change the test back to accessing the content by checkbox.

Reply:
@handeyeco @kevinbarabash Okay, I went through and removed the hidden button references. I exchanged them with references to the radio buttons that I added. Some of my other tests show that clicking on either should produce the same results (except if you try to de-select an option by clicking the radio button; doesn't work because natively that doesn't work). I tried to have it match how it was originally as closely as possible.

Reply:
> except if you try to de-select an option by clicking the radio button; doesn't work because natively that doesn't work

@Myranae so the behavior will be different for sighted users (cannot deselect radio) vs non-sighted users (can deselect radio)?

Reply:
@handeyeco yes, that part will be different. I think we talked about this with Sarah at one point during our design meetings. She said that maybe we should change that for the visual users too as you don't really expect to be able to uncheck a radio button. That's not usually how it works. In the summary for this PR, I discuss this a little. I did find a potential workaround to allow users to uncheck the native radio buttons, but it seemed a bit complex, so didn't think this was the place for that.

Reply:
Okay, I would just double check with Sarah and Jeanette. My understanding is that having different behavior for sighted vs non-sighted users is considered a bad idea. If they say it's fine then NAN; if they say it's acceptable in the short-term, let's write a new ticket.